test/TEST-87-RESOLVED-MDNS/meson.build

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+integration_tests += [
+        integration_test_template + {
+                'name' : fs.name(meson.current_source_dir()),
```

Comment:
@vchillara this needs to set `'vm' : true,` otherwise it will not be able to run nspawn